excel/beginner/basics-of-excel/cells.md

```diff
@@ -0,0 +1,42 @@
+---
+author: Stefan-Stojanovic
+
+levels:
+  - beginner
+
+aspects:
+  - new
+  - workout
+
+type: normal
+
+category: how to
+
+---
+
+# What is a cell?
+
+---
+## Content
+
+A cell in Excel is where a row and a column intersect.
+
+Since each row has a number, and each column has a letter, each cell corresponds to a letter-number combination. This combination is called a cells `address`.

```

Comment:
```suggestion
Since each row has a number, and each column has a letter, each cell corresponds to a letter-number combination. This combination is called a cell's `address`.
```

Reply:
Maybe also add an example like:

> As an example, a cell in column `C` and row `20` has the address `C20`.